arkouda/reductions.py

```diff
@@ -0,0 +1,83 @@
+from arkouda.client import generic_msg, verbose
```

Comment:
this is the third definition of `mink` `maxk`

Reply:
Oops! Thanks for catching that, I think this was a residual file from a name change, I need to look over these branches a little closer before opening the PR's!